src/modules/order.ts

```diff
@@ -181,6 +199,8 @@ export namespace OrderModule {
     /** commission percents, input 10 for 10% */
     referralCommission?: number;
     referralKickback?: number;
+    /** default leverage to 3x for new trades */
```

Comment:
maybe comment like:
leverage is 3x as default. this will help order creator set leverage for market to provided value.